### DISCUSSION TITLE
Just want to ask. what's the diffrence between this and native IMAP in n8n ?

### DISCUSSION THREAD

**Discussion by ARHAEEM** [umanamente/n8n-nodes-imap#4] [Q&A] [answered]
Hey, this is not an issue. just want to know what's the key diffrence between your node and the one in N8N core list.

**Comment by umanamente** [comment, marked as answer]
N8N core node is a trigger. It doesn't provide methods to manipulate emails without receiving one.

**Comment by RayPS** [comment]
The core IMAP node is a [wrapper](https://github.com/n8n-io/imap/blob/master/lib/imapSimple.js) of a very old and outdated version of `node-imap`, some standard IMAPv4 commands are not supported yet such as the very important `ID` extension, it causes a big problem where some mail providers require ID declared to be able to connected via IMAP, otherwise it will be refuse to login, it's similar to "user-agent" but for mailing protocol.

I'm looking for an alternative mail trigger to the core one.

**Reply by RayPS** [reply]
https://imapflow.com/global.html#IdInfoObject

```ts
const client = new ImapFlow({
    clientInfo: {
        name: "n8n-nodes-imap"
    }
})
```
